packaging/win/delugewin.spec

```diff
@@ -19,8 +19,12 @@ hiddenimports += collect_submodules('deluge')
 # Add stdlib as Hidden Imports.
 # This is filtered list that excludes some common examples or stuff not useful in plugins (such as tty, mailbox, tutledemo etc.).
 # It is safe to assume that 90% of that list would already be included anyway.
-stdlib = [ 'pickle', 'json', 'email', 'http', 'marshal', 'uuid', 'base64', 'string', 'struct', 'glob', 'ssl', 'urllib', 'html', 'sys',
-  'crypt', 'datetime', 'hmac', 'locale', 'queue', 're', 'enum', 'collections', 'tokenize', 'hashlib', 'xml', 'csb', 'ipaddress', 'os' ]
+if sys.versioninfo.major >= 3 and sys.versioninfo.minor < 10:
+    stdlib = [ 'pickle', 'json', 'email', 'http', 'marshal', 'uuid', 'base64', 'string', 'struct', 'glob', 'ssl', 'urllib', 'html', 'sys',
+      'crypt', 'datetime', 'hmac', 'locale', 'queue', 're', 'enum', 'collections', 'tokenize', 'hashlib', 'xml', 'csb', 'ipaddress', 'os' ]
+if sys.versioninfo.major >= 3 and sys.versioninfo.minor >= 10:
```

Comment:
This code is broken, on `>= 3 < 10` nothing happens.

Reply:
`version_info` not `versioninfo`